bgpd/bgp_aspath.h

```diff
@@ -32,15 +32,15 @@
 
 /* Private AS range defined in RFC2270.  */
 #define BGP_PRIVATE_AS_MIN       64512U
-#define BGP_PRIVATE_AS_MAX       65535U
+#define BGP_PRIVATE_AS_MAX UINT16_MAX
```

Comment:
can we follow code formating around this spot?

Reply:
Oops

Reply:
don't worry about it.  Let's move forward.  In the grand scheme of the things I am not concerned

Reply:
but yes.  I was actually moving the mouse to hit `submit` when it was merged :)  I was lol'ing

Reply:
Sorry, will take care not to touch formatting next time, I just followed clang-formatting.